src/StackCreate.java

```diff
@@ -0,0 +1,182 @@
+// Operation:           push    pop     top     gtMin
+// Time Complexity:     O(1)    O(1)    O(1)     O(1)
+// Space Complexity:    O(n)    O(n)    O(n)     O(n)
+// Yes, this code ran successfully
+// No, I didn't face any problem in this problem statement in Brute force approach
+
+package src;
+
+import java.util.Stack;
+
+class minStack
+{
+    private int min ;
+    private Stack<Integer> stk ;
+
+    public minStack()
+    {
+        min = Integer.MAX_VALUE ;
+        stk = new Stack<>() ;
+    }
+
+    public void push(int val) {
+        if(min >= val)
+        {
+            stk.push(min) ;                                                     // Pushing extra value(current min)
+            min = val ;
+        }
+        stk.push(val) ;                                                         // Pushing value
+    }
+
+    public void pop() {
+        int temp = stk.pop() ;                                                  // Popping top element
+        System.out.println("Popped item from main stack is " + temp);
+        if(temp == min)                                                         // Popping extra value(old min) pushed earlier
+            min = stk.pop() ;
+    }
+
+    public int top() {
+        return stk.peek() ;                                                     // Getting top element
+    }
+
+    public int getMin() {
+        return min ;                                                            // Getting minimum value
+    }
+    public void printStack()
+    {
+//        System.out.println("********** Printing Stack **********");
+//        stk.forEach(
+//                k -> {
+//                    System.out.println(""+k);
+//                }
+//        );
+        System.out.println(stk);                                                // Printing Stack elements
+    }
+}
+
+public class StackCreate
+{
+    public static void main(String[] args) {
+        minStack s = new minStack();
+        s.push(2);
+        s.printStack();
+        System.out.println("Min val: " + s.getMin());
+        s.push(3);
+        s.printStack();
+        System.out.println("Min val: " + s.getMin());
```

Comment:
You can avoid manual testing by indicating if all the test cases ran successfully or not in leetcode.